its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -901,6 +956,45 @@ private void runCSharpSharedFileWithOneProjectUsingProjectBaseDir(Function<Path,
       .isNotNull();
   }
 
+  private BuildResult runAnalysisWithoutProjectBasedDir(Path projectDir)
+  {
+    String token = TestUtils.getNewToken(ORCHESTRATOR);
+    String folderName = projectDir.getFileName().toString();
+    ScannerForMSBuild scanner = TestUtils.newScanner(ORCHESTRATOR, projectDir, ScannerClassifier.NET_5)
+      .addArgument("begin")
+      .setProjectKey(folderName)
+      .setProjectName(folderName)
+      .setProjectVersion("1.0")
+      // do NOT set "sonar.projectBaseDir" for this test
+      .setProperty("sonar.login", token)
+      .setUseDotNetCore(Boolean.TRUE)
```

Comment:
Do we really need to set this flag when the `ScannerClassifier` is already `NET_5`? It looks weird. If yes, it's probably a bug on Orchestrator side

Reply:
it's our wrappers, thanks.